### ISSUE TITLE
Method descriptor

### ISSUE THREAD

**Issue by kushal256** [javaparser/javasymbolsolver#141]
Is there a way to get the method descriptor for a method (see http://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.3.3 ).  JavaSymbolSolver would be great to resolve the fully qualified objects wondering if anyone already has written something like this?

Thanks!

**Comment by ftomassetti** [comment]
No, but we have some code to do the opposite thing because we work also with compiled code and from compiled code we want to derive readable descriptors.

That said it should not be too hard to implement. It is just not a top priority right now (but I would love to receive a Pull Request for this).
